run/map/watermark/guideposts.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "assetName":"test.png",
+    "src":"/static/run/map/watermark/test.png",
+    "imageRef":"watermark",
+    "srcX":0,
+    "srcY":0,
+    "srcHeight":634,
+    "srcWidth":634,
+    "dWidth":200,
+    "dHeight":200,
+    "mapCoord":{
+      "mapName":"world1",
+      "x":10,
+      "y":1
+    },
+    "position":1
```

Comment:
As I commented above, using string literals such as `topleft` or `center` is more readable than `1` or `5`.